### DISCUSSION TITLE
Meaning of "Expunged notes/notebooks" on first run

### DISCUSSION THREAD

**Discussion by nikbpetrov** [vzhd1701/evernote-backup#35] [General]
Hi, many thanks for this repo - really amazing stuff!

Just one question regarding the "expunged notes/notebooks" log on first run. I got 7 notebooks and 21 notes. This scared me for first run and got me [checking the doc](https://dev.evernote.com/doc/reference/NoteStore.html#Fn_NoteStore_expungeNotebook) to ensure that 3rd party apps can't really delete stuff from my account.

I figured, after also checking the code, that this is "expunged from .db" rather than expunged from my account. I tested it out with a new note and that checked out - after creating a note and syncing, 1 note was updated and after expunging that note from my account manually, the note was expunged from the .db file.

Though, still I can't quite crack why I got 7 notebooks and 21 notes on first syncing. Would that be a count of historical notes/notebooks I have expunged in my account? Though that wouldn't make sense as the meaning of "expunged" would change depending on the state of the .db which is a bit funny.

**Comment by vzhd1701** [comment]
Notes are expunged when they are deleted from the Evernote server. During the sync, evernote-backup [receives lists of expunged IDs](https://github.com/vzhd1701/evernote-backup/blob/e09a805b8187aa229b7cf70ce69fd1225a347626/evernote_backup/note_synchronizer.py#L298) from [Evernote server](https://dev.evernote.com/doc/reference/NoteStore.html#Struct_SyncChunk) and wipes those out from the local database file.

During the first sync Evernote server sends all previously expunged IDs from the very beginning IIRC.

Also, total number of synced notes may include trash and notebooks that you have shared access to.
